ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -85,6 +92,20 @@ module.exports = Marionette.LayoutView.extend({
             model: this.model
         }));
     },
+    showResult: function () {
+        this.$el.trigger('closeDropdown.'+CustomElements.getNamespace());
+        lightboxInstance.model.updateTitle(this.model.get('title'));
+        lightboxInstance.model.open();
+        lightboxInstance.$el.addClass('is-result-form');
```

Comment:
Is this not handled already by [this](https://github.com/codice/ddf/pull/3192/files#diff-00f64a898397aba336167bc43a86355dR20)?